src/credential/issuance/__tests__/04-complete-user-authorization.test.ts

```diff
@@ -1,48 +1,23 @@
-import type { AuthorizationContext } from "src/utils/auth";
 import {
   AuthorizationError,
   AuthorizationIdpError,
 } from "../../../utils/errors";
-import { completeUserAuthorizationWithQueryMode } from "../04-complete-user-authorization";
-import { CredentialIssuerEntityConfiguration } from "../../../trust/types";
+import { parseAuthroizationResponse } from "../04-complete-user-authorization";
+import parseUrl from "parse-url";
 
 describe("authorizeUserWithQueryMode", () => {
-  // Mocks required for the authorization process
-  const issuerRequestUri = "https://example.com/authz";
-  const redirectUri = "test://cb";
-  const idpHint = "idp";
-  const clientId = "clientId";
-  const issuerConf = {
-    oauth_authorization_server: {
-      authorization_endpoint: "https://example.com/auth",
-    }, // This is the only field required for the test
-  } as CredentialIssuerEntityConfiguration["payload"]["metadata"];
-
   it("should return the authorization result when the authorization server responds with a valid response", async () => {
     const authRes = {
       code: "abcdefg",
       state: "123456",
       iss: "123456",
     };
+    const authRedirectUrl = `test://cb?code=abcdefg&state=123456&iss=123456`;
 
-    const authContext: AuthorizationContext = {
-      authorize: jest
-        .fn()
-        .mockResolvedValue(
-          `test://example.com?${new URLSearchParams(authRes).toString()}`
-        ),
-    };
-
-    const result = await completeUserAuthorizationWithQueryMode(
-      issuerRequestUri,
-      clientId,
-      issuerConf,
-      idpHint,
-      redirectUri,
-      authContext
-    );
+    const query = parseUrl(authRedirectUrl).query;
+    const authResParsed = parseAuthroizationResponse(query);
```

Comment:
Is there a reason we are not testing `completeUserAuthorizationWithQueryMode` but only `parseAuthroizationResponse`?